com.milesalan.mepo.yml

```diff
@@ -0,0 +1,123 @@
+# Flatpak Manifest
+app-id: com.milesalan.mepo
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.ziglang
+rename-icon: mepo
+rename-desktop-file: mepo.desktop
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --share=network
+  - --filesystem=xdg-documents
+
+build-options:
+  append-path: /usr/lib/sdk/ziglang
+command: mepo
+modules:
+  # Core build deps for mepo (note SDL2 is in runtime)
+  - name: SDL2_image
```

Comment:
both SDL2_image and SDL2_ttf are in the runtime too. Albeit version 2.0.5 and 2.0.15 respectively.